tests/integration/tests/test-breaches-page.js

```diff
@@ -0,0 +1,15 @@
+/* global browser */
+
+"use strict";
+
+describe("Firefox Monitor Breaches Page", function() {
+  this.retries(2);
+
+  beforeEach(function() {
+    browser.url("http://localhost:6060/breaches");
```

Comment:
Since we already define the `baseUrl` in the configs, do we need the fully qualified URL here? Or can this just be a vague "/breaches", which infers the domain from the config?

Reply:
Well the baseurl doesn't do much. I could just do ```browser.url(global.baseUrl)```. There's not really any standard on doing baseurl with wdio.

Reply:
Well I guess I should read. Seems they have baseUrl support but maybe it didn't work before. I was probably doing it wrong..

Reply:
I am not a Selenium/wdio expert, but my understanding was that if you passsed a relative URL, it'd use the `baseUrl` from the config. So basically, could we use `browser.url("/breaches")` and have it infer the `baseUrl` from the config. But really, I should be pulling the PR locally and testing that theory.